.github/scripts/gmp_build_and_test.sh

```diff
@@ -15,4 +15,6 @@ cmake \
   .. && \
 make -j 2 && \
 make install && \
+ctest -R test_gmp1 -V
```

Comment:
why not just 
```
ctest --output-on-failure
```
Which runs all the tests?

(Otherwise there are missing `&& \` to get the failure if a test fails.)

Reply:
agreed; before I was running one particular test, but now we just run all of them.